web/robots.txt

```diff
@@ -1,5 +1,5 @@
 # www.robotstxt.org/
-# www.google.com/support/webmasters/bin/answer.py?hl=en&answer=156449
+# support.google.com/webmasters/answer/6062608
```

Comment:
Let's use https?

Reply:
Yes

Reply:
Could you update the PR?
`https://support.google.com/webmasters/answer/6062608c`
thx

Reply:
Then I'd prefer to use HTTPS on the previous link as well, i.e. `https://www.robotstxt.org/`